-------------------------------------------------------------------------------
Test set: models.dao.Sql2oNewsDaoTest
-------------------------------------------------------------------------------
Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 1.749 sec <<< FAILURE! - in models.dao.Sql2oNewsDaoTest
models.dao.Sql2oNewsDaoTest  Time elapsed: 1.749 sec  <<< ERROR!
org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "41.84.152.6", user "rbwqysqbdiaqms", database "d9stjanqk4t4gk", SSL off
	at models.dao.Sql2oNewsDaoTest.setUp(Sql2oNewsDaoTest.java:30)
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "41.84.152.6", user "rbwqysqbdiaqms", database "d9stjanqk4t4gk", SSL off
	at models.dao.Sql2oNewsDaoTest.setUp(Sql2oNewsDaoTest.java:30)

models.dao.Sql2oNewsDaoTest  Time elapsed: 1.749 sec  <<< ERROR!
java.lang.NullPointerException
	at models.dao.Sql2oNewsDaoTest.shutDown(Sql2oNewsDaoTest.java:43)

